### PR TITLE
Docker

### PR DESCRIPTION
**Pull request by ousamamhadden**

### CODE REVIEW

File: dockerfiles/predict_model.dockerfile [hunk 1]
```diff
@@ -5,13 +5,16 @@ RUN apt update && \
     apt install --no-install-recommends -y build-essential gcc && \
     apt clean && rm -rf /var/lib/apt/lists/*
 
+#We install requirements before any copying for better caching 
 COPY requirements.txt requirements.txt
-COPY pyproject.toml pyproject.toml
```
Comment (alanmansour) [inline dockerfiles/predict_model.dockerfile:9]: Is this not needed?

File: dockerfiles/train_model.dockerfile [hunk 2]
```diff
@@ -5,13 +5,14 @@ RUN apt update && \
     apt install --no-install-recommends -y build-essential gcc && \
     apt clean && rm -rf /var/lib/apt/lists/*
 
+#We install requirements before any copying for better caching 
 COPY requirements.txt requirements.txt
-COPY pyproject.toml pyproject.toml
```
Comment (alanmansour) [inline dockerfiles/train_model.dockerfile:9]: Is this not needed?

**Review by alanmansour — CHANGES_REQUESTED** [review]

File: dockerfiles/predict_model.dockerfile (hunk 1 above)
Comment (ousamamhadden) [reply dockerfiles/predict_model.dockerfile:9]: It works without it. Maybe we need it later for something!?! We can add it back if needed later or I can add it back now I got no preferences.

Comment (ousamamhadden) [reply dockerfiles/predict_model.dockerfile:9]: you decide :PP

Comment (alanmansour) [reply dockerfiles/predict_model.dockerfile:9]: I know make commands fail if smth is wrong or missing in pyproject.toml, confirmed this yesterday hehe

Comment (ousamamhadden) [reply dockerfiles/predict_model.dockerfile:9]: Aight then I am putting it back in :)

**Review by alanmansour — APPROVED** [review]

**Pull request merged**